vtexdocs/devportal
# fix(header): update header style

#### What is the purpose of this pull request?

Update the header style with the new Figma design.

#### What problem is this solving?

The header is out of date compared with the Figma version. This PR proposes that update.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-58--elated-hoover-5c29bf.netlify.app/) and compare the Header style with [this page in Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=4088%3A253765) and check the correct [prototype flow here](https://www.figma.com/proto/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=4088%3A253765&scaling=min-zoom&page-id=129%3A28&starting-point-node-id=2911%3A163480&hide-ui=1).

#### Screenshots or example usage

https://user-images.githubusercontent.com/42784961/178616463-fe022796-3ff6-4783-8038-2096f9f25a23.mp4

#### Types of changes
